gslib/gcs_json_api.py

```diff
@@ -326,7 +326,8 @@ def _ValidateEncryptionFieldsForMetadataGetRequest(self, fields=None):
 
   def GetBucketIamPolicy(self, bucket_name, provider=None, fields=None):
     apitools_request = apitools_messages.StorageBucketsGetIamPolicyRequest(
-        bucket=bucket_name, userProject=self.user_project)
+        bucket=bucket_name, userProject=self.user_project,
+        requestedPolicyVersion=3)
```

Comment:
Should this value be configurable now or in the future? I saw the doc mentioned the value being "at least 3", but I don't have a lot of background on this, so I wasn't sure if a user would ever actually need to set this greater than 3.

Reply:
I think adding configuration here would only pollute the configuration file as this will likely only change once in a blue moon. If this situation ever changes we can obviously revisit this.

Reply:
Resolved this as I just noticed there's a note saying this doesn't need to be configurable in the doc you linked to earlier.